Jenkinsfile

```diff
@@ -48,22 +48,24 @@ def buildManifest = {String manifest, String bitbake_image ->
         sh "vagrant ssh -c \"/vagrant/vagrant-cookbook/yocto/build-images.sh ${yoctoDir} ${bitbake_image}\""
     }
 
-    stage("Copy images and cache ${bitbake_image}") {
-        sh "rm -rf archive"
-        sh "mkdir -p archive"
-        sh "vagrant ssh -c \"cp -a ${yoctoDir}/build/tmp/deploy/images/ /vagrant/archive/\""
-
-        // Archive the downloads and sstate when the environment variable was set to true
-        // by the Jenkins job.
-        if (env.ARCHIVE_CACHE) {
-            sh "vagrant ssh -c \"cp -a ${yoctoDir}/build/downloads/ /vagrant/archive/\""
-            sh "vagrant ssh -c \"cp -a ${yoctoDir}/build/sstate-cache/ /vagrant/archive/\""
-        }
-    }
-
-    stage("Archive ${bitbake_image}") {
-        archiveArtifacts artifacts: "archive/**/*"
-    }
+// Use this again when we can give out complete and corresponding source code as well
```

Comment:
Keeping this for future reference? I don't mind much but I usually get a slap in the face when doing this ;)

Reply:
I say remove it. When we want it back we can just revert the commit.
That is one of the use-cases with having atomic commits :)

Reply:
@DunderRoffe -<---@

Reply:
Good point, I'll remove it completely instead.